openrtb_ext/seat_non_bids.go

```diff
@@ -0,0 +1,78 @@
+package openrtb_ext
+
+import "github.com/prebid/openrtb/v19/openrtb2"
+
+// NonBidsWrapper contains the map of seat with list of nonBids
+type NonBidsWrapper struct {
+	seatNonBidsMap map[string][]NonBid
+}
+
+// NonBidParams contains the fields that are required to form the nonBid object
+type NonBidParams struct {
```

Comment:
rename it to `nonbid`

Reply:
we can't use nonbid because we need to expose it outside the package.
To make it public we will need any other name because `NonBid` is already decleared in same package